src/main/cpp/t04_deposit.cpp

```diff
@@ -22,5 +22,17 @@
 using namespace std;
 
 int t04_deposit() {
-
+    int p, x, y, k, i = 0, y0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'y0' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4716804)